apps/epl_ets/src/epl_ets.erl

```diff
@@ -0,0 +1,187 @@
+%% API functions
+%%====================================================================
+
+%% @doc Stars epl_ets.
+-spec start_link() -> {ok, Pid :: pid()} |
+                      ignore |
+                      {reason, Reason :: term()}.
+start_link() ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+
+%% @doc Add calling process to epl_ets subscribers.
+-spec subscribe() -> ok.
+subscribe() ->
+    subscribe(self()).
+
+%% @doc Add provided `Pid` to epl_ets subscribers.
+-spec subscribe(Pid :: pid()) -> ok.
+subscribe(Pid) ->
+    gen_server:cast(?MODULE, {subscribe, Pid}).
+
+%% @doc Remove calling process from epl_ets subscribers.
+-spec unsubscribe() -> ok.
+unsubscribe() ->
+    unsubscribe(self()).
+
+%% @doc Remove provided `Pid` from epl_ets subscribers.
+-spec unsubscribe(Pid :: pid()) -> ok.
+unsubscribe(Pid) ->
+    gen_server:cast(?MODULE, {unsubscribe, Pid}).
+
+%%====================================================================
+%% gen_server callbacks
+%%====================================================================
+
+init([]) ->
+    Res = epl:subscribe(),
+    true = if_subs_ok(Res),
+    check_node_timer(),
+    Node = epl:get_default_node(),
+    VizEntity = epl_viz_map:new(Node),
+    {ok, #state{vizceral = VizEntity}}.
+
+handle_call(Request, _From, _State) ->
+    exit({not_implemented, Request}).
+
+handle_cast({subscribe, Pid}, State = #state{subscribers = Subs}) ->
+    NewSubs = [Pid | Subs],
+    NewState = State#state{subscribers = NewSubs},
+    {noreply, NewState};
+handle_cast({unsubscribe, Pid}, State = #state{subscribers = Subs}) ->
+    NewSubs = lists:delete(Pid, Subs),
+    NewState = State#state{subscribers = NewSubs},
+    {noreply, NewState}.
+
+handle_info({data, {Node, _Timestamp}, _Proplist},
+            State = #state{subscribers = Subs, vizceral = VizEntity,
+                          nodes = SubsNodes}) ->
+    NewSubsNodes = register_node_activity(Node, SubsNodes),
+    NewViz = update_viz(Node, VizEntity, SubsNodes),
+    distribute_viz(NewViz, Subs),
+    NewState = State#state{vizceral = NewViz, nodes = NewSubsNodes},
+    {noreply, NewState};
+handle_info(check_nodes_state, State = #state{vizceral = VizEntity,
+                                                nodes = SubsNodes}) ->
+    NewSubsNodes = remove_outdated(SubsNodes),
+    NewVizEntity = remove_outdated(NewSubsNodes, VizEntity),
+    check_node_timer(),
+    NewState = State#state{vizceral = NewVizEntity, nodes = NewSubsNodes},
+    {noreply, NewState}.
+
+terminate(_Reason, _State) ->
+    epl:unsubscribe().
+
+code_change(_OldVsn, State, _Extra) ->
+    {ok, State}.
+
+%%====================================================================
+%% Internals
+%%====================================================================
+
+get_ets_basic_info(Node) ->
+    ETSCount = get_all_ets_count(Node),
+    ETSMemUsage = get_ets_mem_usage(Node),
```

Comment:
Maybe instead of calling erlang:memory/0 through epl_tracer:command(), change epl_tracer:handle_info() so that it collects all memory statistics.